.tito/packages/mock

```diff
@@ -1 +1 @@
-2.16-1 mock/
+3-1 mock/
```

Comment:
This should be `3.0`, not `3`

Reply:
Why?

Reply:
I see:
```
rpmdev-vercmp 3-1 3.0-0
3-1 < 3.0-0
```
And we have added `Requires: mock >= 3.0` in fedora-review :-(